metricbeat/schema/mapstriface/mapstriface.go

```diff
@@ -71,18 +72,22 @@ type ConvMap struct {
 }
 
 // Map drills down in the data dictionary by using the key
-func (convMap ConvMap) Map(key string, event common.MapStr, data map[string]interface{}) {
+func (convMap ConvMap) Map(key string, event common.MapStr, data map[string]interface{}) multierror.Errors {
 	subData, ok := data[convMap.Key].(map[string]interface{})
 	if !ok {
 		if !convMap.Optional {
-			logp.Err("Error accessing sub-dictionary `%s`", convMap.Key)
+			err := fmt.Errorf("Error accessing sub-dictionary `%s`", convMap.Key)
+			logp.Err("%s", err)
+			errs := multierror.Errors{}
+			errs = append(errs, err)
+			return errs
```

Comment:
Could these lines be written as `multierror.Errors{err}`? Would be nicer if it works.